### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Position of the radiobuttons are incorrectly announced by Jaws and NVDA in "Azure Sign in dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1459]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to interactive and automated radio buttons.
4. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
Jaws and NVDA are not announcing the position of the radio buttons correctly

**Jaws Behaviour:**
Interactive radio button checked
2 of 2
To change the selection press up or down arrow
Automated radio button not checked
1 of 2 
To change the selection press up or down arrow

**NVDA Behaviour:**
Interactive  radio button  checked  2 of 2
Automated  radio button  not checked  1 of 2

**Expected Result:**
Jaws and NVDA should announce the position of radio buttons correctly

Jaws should announce as
Interactive radio button checked
1 of 2
To change the selection press up or down arrow
Automated radio button not checked
2 of 2 
To change the selection press up or down arrow

NVDA should announce as
Interactive  radio button  checked  1 of 2
Automated  radio button  not checked  2 of 2

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![sign in-radio buttons](https://user-images.githubusercontent.com/22956130/36099813-076d8e38-102a-11e8-93d1-19083d32865e.jpg)
![fa5235ed](https://user-images.githubusercontent.com/22956130/36099817-0aae45ce-102a-11e8-94b8-c8e7c0a95ec8.jpg)
![309a9360](https://user-images.githubusercontent.com/22956130/36099821-0f1cb802-102a-11e8-9af2-7abf615afdbe.jpg)

**Comment by Eskibear** [comment]
According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749933
